client/homescreen/test/index.js

```diff
@@ -268,26 +267,26 @@ describe( 'Homescreen Layout', () => {
 		const secondColumn = columns[ 1 ];
 
 		expect(
-			within( firstColumn ).getByText( /\[TaskList\]/ )
+			within( firstColumn ).getByText( '[TaskList]' )
 		).toBeInTheDocument();
 		expect(
-			within( firstColumn ).getByText( /\[InboxPanel\]/ )
+			within( firstColumn ).getByText( '[InboxPanel]' )
 		).toBeInTheDocument();
 		expect(
-			within( secondColumn ).queryByText( /\[TaskList\]/ )
-		).toBeNull();
+			within( secondColumn ).queryByText( '[TaskList]' )
+		).not.toBeInTheDocument();
 		expect(
-			within( secondColumn ).queryByText( /\[InboxPanel\]/ )
-		).toBeNull();
+			within( secondColumn ).queryByText( '[InboxPanel]' )
+		).not.toBeInTheDocument();
 
 		expect(
-			within( secondColumn ).getByText( /\[StatsOverview\]/ )
+			within( secondColumn ).getByText( '[StatsOverview]' )
```

Comment:
I went for text instead of the regex, because its less to type and easier to edit later.